File: ruby/01/day-one.rb
#!/usr/bin/env ruby

require 'timeout'

i = 4
d = 4.0
s = "Run's house "

j = 0
e = 0.0
t = ""

begin
  Timeout::timeout(1) do
    j = gets

    k = i + j.to_i

    puts k
  end

  Timeout::timeout(1) do
    e = gets
  end

  Timeout::timeout(1) do
    t = gets
  end

rescue Timeout::Error
  puts "There weren't no input."
end

File: py/01/day-one.py
#!/usr/bin/env python

from threading import Timer

i = 4
d = 4.0
s = "Run's house "

a = Timer(1, print, ["That weren't no input."])
a.start()
j = input()
a.cancel()

b = Timer(1, print, ["That weren't no input."])
b.start()
e = input()
b.cancel()

c = Timer(1, print, ["That weren't no input."])
c.start()
t = input()
c.cancel()


try: 
    k = i + int(j)
    print(k)
except Exception as exc:
    print(exc)
